src/Core/CMakeLists.txt

```diff
@@ -37,10 +71,14 @@ else()
   set (OpenMP_EXE_LINKER_FLAGS ${OpenMP_C_FLAGS})
 endif()
 
-target_link_libraries(cilacc ${OpenMP_EXE_LINKER_FLAGS} )
-include_directories(cilacc PUBLIC 
-				  ${CMAKE_CURRENT_SOURCE_DIR}/include
-		          )
+
```

Comment:
I'd suggest to move the block about OpenMP just after OpenMP is found and merge the `add_library` and `include_directories` and `target_link_libraries` commands.